<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="RuntimeWrappedException.xml" source-language="en-US" target-language="es-ES">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5a98ca6a9d59dbbe9f80b9124e4f2798b489d5969.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">a98ca6a9d59dbbe9f80b9124e4f2798b489d5969</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>Wraps an exception that does not derive from the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> class.</source>
          <target state="translated">Ajusta una excepción que no deriva de la clase <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>This class cannot be inherited.</source>
          <target state="translated">Esta clase no puede heredarse.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>Some languages, such as C++, allow you to throw exceptions of any managed type.</source>
          <target state="translated">Algunos lenguajes, como C++, permiten iniciar excepciones de cualquier tipo administrado.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>Other languages, such as Microsoft C# and Visual Basic, require that every thrown exception be derived from the <ph id="ph1">&lt;xref:System.Exception&gt;</ph> class.</source>
          <target state="translated">Otros lenguajes, como Microsoft C# y Visual Basic, requieren que cada excepción iniciada se deriva de la <ph id="ph1">&lt;xref:System.Exception&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>To maintain compatibility between languages, the common language runtime (CLR) wraps objects that do not derive from <ph id="ph1">&lt;xref:System.Exception&gt;</ph> in a <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException&gt;</ph> object.</source>
          <target state="translated">Para mantener la compatibilidad entre idiomas, common language runtime (CLR) ajusta los objetos que no se derivan de <ph id="ph1">&lt;xref:System.Exception&gt;</ph> en un <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException&gt;</ph> objeto.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>You can use the <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute&gt;</ph> class to specify whether exceptions should appear wrapped inside catch blocks and exception filters for an assembly.</source>
          <target state="translated">Puede usar el <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute&gt;</ph> clase para especificar si las excepciones deben aparecer ajustadas dentro de bloques catch y filtros de excepción para un ensamblado.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>Many language compilers, including the Microsoft C# and Visual Basic compilers, apply this attribute by default to specify the wrapping behavior.</source>
          <target state="translated">Muchos de los compiladores de lenguaje, incluidos los compiladores de Microsoft C# y Visual Basic, aplican este atributo para especificar el comportamiento de ajuste de forma predeterminada.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>Note that the runtime still wraps exceptions even if you use the <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute&gt;</ph> class to specify that you do not want them wrapped.</source>
          <target state="translated">Tenga en cuenta que el runtime las excepciones ajusta incluso si utiliza la <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute&gt;</ph> clase para especificar que no desea ajustada.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>In this case, exceptions are unwrapped only inside catch blocks or exception filters.</source>
          <target state="translated">En este caso, las excepciones son ajustadas solo dentro de los bloques catch o filtros de excepciones.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.CompilerServices.RuntimeWrappedException">
          <source>The following code example demonstrates how to throw a <ph id="ph1">&lt;xref:System.String&gt;</ph> object as an exception in C++ and catch it using a <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException&gt;</ph> object.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra cómo producir una <ph id="ph1">&lt;xref:System.String&gt;</ph> objeto como una excepción de C++ y detectarla utilizando un <ph id="ph2">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException&gt;</ph> objeto.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Objeto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> que contiene los datos serializados del objeto relativos a la excepción que se va a producir.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> object that contains contextual information about the source or destination.</source>
          <target state="translated">El objeto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> que contiene información contextual sobre el origen o el destino.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Sets the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object with information about the exception.</source>
          <target state="translated">Establece el objeto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> con información sobre la excepción.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData%2A&gt;</ph> method sets a <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> object with all the exception object data targeted for serialization.</source>
          <target state="translated">El <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData%2A&gt;</ph> método establece una <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> objeto con todos los datos del objeto de excepción destinados a la serialización.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During deserialization, the exception is reconstituted from the <ph id="ph1">`SerializationInfo`</ph> transmitted over the stream.</source>
          <target state="translated">Durante la deserialización, la excepción se reconstituye a partir de la información <ph id="ph1">`SerializationInfo`</ph> transmitida en la secuencia.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;paramref name="info" /&gt;</ph> parameter is <ph id="ph2">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">El valor del parámetro <ph id="ph1">&lt;paramref name="info" /&gt;</ph> es <ph id="ph2">&lt;see langword="null" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>requires full trust for the immediate caller.</source>
          <target state="translated">requiere plena confianza para el llamador inmediato.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.CompilerServices.RuntimeWrappedException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This member cannot be used by partially trusted or transparent code.</source>
          <target state="translated">Este miembro no puede usarse por código de confianza parcial o transparente.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="P:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException">
          <source>Gets the object that was wrapped by the <ph id="ph1">&lt;see cref="T:System.Runtime.CompilerServices.RuntimeWrappedException" /&gt;</ph> object.</source>
          <target state="translated">Obtiene el objeto ajustado por el objeto <ph id="ph1">&lt;see cref="T:System.Runtime.CompilerServices.RuntimeWrappedException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException">
          <source>The object that was wrapped by the <ph id="ph1">&lt;see cref="T:System.Runtime.CompilerServices.RuntimeWrappedException" /&gt;</ph> object.</source>
          <target state="translated">Objeto ajustado por el objeto <ph id="ph1">&lt;see cref="T:System.Runtime.CompilerServices.RuntimeWrappedException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException">
          <source>The <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException%2A&gt;</ph> property gets an object that was thrown as an exception from a language that allows exceptions that do not derive from the <ph id="ph2">&lt;xref:System.Exception&gt;</ph> class.</source>
          <target state="translated">El <ph id="ph1">&lt;xref:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException%2A&gt;</ph> propiedad obtiene un objeto que se inicia una excepción desde un lenguaje que permite que las excepciones que no se derivan de la <ph id="ph2">&lt;xref:System.Exception&gt;</ph> clase.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>